schutzbot/deploy.sh

```diff
@@ -146,7 +146,8 @@ echo -e "fastestmirror=1" | sudo tee -a /etc/dnf/dnf.conf
 sudo dnf -y install jq
 
 # Get latest commit from osbuild-composer main branch
-GIT_COMMIT=$(get_last_passed_commit)
+# GIT_COMMIT=$(get_last_passed_commit)
+GIT_COMMIT='d71574976ee2d954f8e04bd3bae9164e46a08a7b'
```

Comment:
Why are you hardcoding this now? `rpmbuild` stage is running on the new runners and is merged so latest main will have the composer package for the new nightlies.

Reply:
Thanks, it was a mistake. I hardcoded it to test it out. I forgot to revert that part back.